### DISCUSSION TITLE
Store level-defaults together with adventures

### DISCUSSION THREAD

**Discussion by TiBiBa** [hedyorg/hedy#2373] [Implementation discussions]
Currently we have two main sources for our code examples:

- Level-defaults (which also includes the commands dropdown)
- Adventures

With the fixing of #2371 we separate the commands from the level-defaults as they are conceptually different. The commands "try it" window is no longer as prominent in our UI as it was before we remove the sidebar. After this splitsing we are left with three different main components:

- Level-defaults
- Commands
- Adventures

Conceptually the level-defaults and adventures are the same: We provide the user with a short intro / explanation, combined with a code snippet or example code. They are able to save program with level name (level-defaults) or adventure name (adventures). The current tab structure is overly complex due to the effort to merge these two files together into one usable UI.
This can be simplified by adding the level-defaults as a seperate "start" adventure to the adventures files, removing the level-defaults structure all together. This will simplify our back-end significantly and also simplify the work for our translators.

Let me know what you think!

**Comment by Felienne** [comment]
Fixed by #2428
